.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous Integration
+on:
+  workflow_run:
+    workflows: [Generate default_config.zip]
```

Comment:
if the only source of trigger for this workflow is "Generate default_config.zip" then this can very well be just another trigger job in "Generate default_config.zip" itself

Reply:
Sure! will put it in the same workflow